src/api/models/sql_models/base.py

```diff
@@ -232,6 +232,9 @@ class BaseSQLModel(models.Model):
         string_agg(DISTINCT visibility_name, ', ' ORDER BY visibility_name) AS visibility_name
     """
 
+    # top-level benthic categories to exclude from SU- and SE-level summaries
```

Comment:
Do you expect that there will be more than one category to exclude?  If not, including `AND benthic_category != 'Other'` in the 3 protocols sql directly is clearer and you wouldn't have to introduce a base class property that isn't being used by the other protocols